app/src/main/java/org/elasticsearch/bootstrap/BootstrapProxy.java

```diff
@@ -182,24 +182,14 @@ private void setup(boolean addShutdownHook, Environment environment) throws Boot
 
         try {
             // look for jar hell
-            final Logger logger = ESLoggerFactory.getLogger(JarHell.class);
+            final Logger logger = LogManager.getLogger(JarHell.class);
             JarHell.checkJarHell(logger::debug);
         } catch (IOException | URISyntaxException e) {
             throw new BootstrapException(e);
         }
 
         IfConfig.logIfNecessary();
 
-        /*
-         * DISABLED setup of security manager due to policy problems with plugins and dependencies.
-         */
-        // install SM after natives, shutdown hooks, etc.
-        //try {
-        //    Security.configure(environment, BootstrapSettings.SECURITY_FILTER_BAD_DEFAULTS_SETTING.get(settings));
-        //} catch (IOException | NoSuchAlgorithmException e) {
-        //    throw new BootstrapException(e);
-        //}
```

Comment:
I'd prefer leaving it commented out, so everyone knows why this differs from ES Bootstrap and why it is disabled

Reply:
The class already has other differences - I'm not sure if maintaining a commented-out mirror is less confusing as removing the things that don't apply to us. Having a mix of it is definitely not helpful and removing this was less work.